packages/xstate-inspect/src/types.ts

```diff
@@ -19,6 +19,7 @@ export interface InspectorOptions {
   iframe?: MaybeLazy<HTMLIFrameElement | null | false>;
   devTools?: MaybeLazy<XStateDevInterface>;
   serialize?: Replacer | undefined;
+  targetWindow?: Window | undefined | null;
```

Comment:
Any reason to include `null` here?
```suggestion
  targetWindow?: Window | undefined;
```

Reply:
TypeScript intelligence reports that `window.open` can return `Window | null`.

```typescript
inspect({
  url,
  // This will throw a type error if targetWindow doesn't support `null` type
  targetWindow: window.open(url)
});
```